hs_core/views/resource_folder_hierarchy.py

```diff
@@ -290,17 +292,20 @@ def data_store_folder_unzip(request, **kwargs):
         return HttpResponse(str(ex), status=status.HTTP_400_BAD_REQUEST)
 
     overwrite = request.POST.get('overwrite', 'false').lower() == 'true'  # False by default
+    auto_aggregate = request.POST.get('auto_aggregate', 'true').lower() == 'true'  # False by default
```

Comment:
Comment looks to be wrong, should be "True by default"

Reply:
thanks for catching that.